numpy/core/src/multiarray/scalartypes.c.src

```diff
@@ -2058,7 +2068,7 @@ static PyMethodDef gentype_methods[] = {
         METH_VARARGS | METH_KEYWORDS, NULL},
     /* Hook for the round() builtin */
     {"__round__",
-        (PyCFunction)gentype_round,
+        (PyCFunction)dunder_round,
```

Comment:
```suggestion
        (PyCFunction)gentyp_dunder_round,
```
Or perhaps
```suggestion
        (PyCFunction)gentype___round__,
```

Reply:
It's not generated though -- Would that still be appropriate?

Reply:
`gentype_something` refers to `np.generic.something` in this file